plugins/reporters/spdx/src/main/kotlin/Extensions.kt

```diff
@@ -42,6 +46,17 @@ import org.ossreviewtoolkit.utils.spdx.model.SpdxPackageVerificationCode
 import org.ossreviewtoolkit.utils.spdx.toSpdx
 import org.ossreviewtoolkit.utils.spdx.toSpdxId
 
+/**
+ * Convert an ORT [Hash] to an [SpdxChecksum], or return null if a conversion is not possible.
+ */
+private fun Hash.toSpdxChecksum(): SpdxChecksum? =
+    SpdxChecksum.Algorithm.values().find { it.name in algorithm.aliases }?.let {
```

Comment:
Would it make sense to add a code comment like: `// Assume that algorithm aliases in ORT and SPDX match.`?

Reply:
I'll do it in a follow-up PR.

Reply:
See https://github.com/oss-review-toolkit/ort/pull/7085.